src/main/java/io/supertokens/webserver/api/totp/CreateOrUpdateTotpDeviceAPI.java

```diff
@@ -4,15 +4,16 @@
 import java.security.NoSuchAlgorithmException;
 
 import com.google.gson.JsonObject;
-
 import io.supertokens.Main;
 import io.supertokens.pluginInterface.RECIPE_ID;
 import io.supertokens.pluginInterface.exceptions.StorageQueryException;
 import io.supertokens.pluginInterface.totp.TOTPDevice;
 import io.supertokens.pluginInterface.totp.exception.DeviceAlreadyExistsException;
 import io.supertokens.pluginInterface.totp.exception.TotpNotEnabledException;
 import io.supertokens.pluginInterface.totp.exception.UnknownDeviceException;
```

Comment:
add a comment in the places where you are doing user id mapping explaining that its being done because of last active user table needs to have internal user ids only